packaging/docker/README.md

```diff
@@ -246,6 +246,13 @@ If you don't want to destroy and recreate your container, you can edit the Agent
 above section on [configuring Agent containers](#configure-agent-containers) to find the appropriate method based on
 how you created the container.
 
+### Custom agent UID/GID
+
+By default, the Netdata Agent in the container will run with a user ID and group ID of 201, matching the default
```

Comment:
```suggestion
By default, Netdata in the container will run with a user ID and group ID of `201`, matching the default
```

Reply:
I’m curious about the rationale behind this change. It’s specifically the agent that’s running in the container, and I was under the impression that we generally prefer to say ‘the Netdata Agent’ in such cases over just ‘Netdata’.